presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -83,6 +83,7 @@
     public static final String FORCE_SINGLE_NODE_OUTPUT = "force_single_node_output";
     public static final String FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_SIZE = "filter_and_project_min_output_page_size";
     public static final String FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_ROW_COUNT = "filter_and_project_min_output_page_row_count";
+    public static final String DYNAMIC_PARTITION_PRUNING = "dynamic_partition_pruning";
```

Comment:
If we go `dynamic_filter(...)` expression way, then whole dynamic filtering pushdown won't be needed because it will be automatically supported by `PredicatePushdown`.

Reply:
same as above comment. this flag is being used for enabling entire DF feature. needs rename.